detection_rules/packaging.py

```diff
@@ -135,6 +136,22 @@ def _package_notice_file(save_dir):
             lines = lines + commented_notice + [' */', '']
             f.write('\n'.join(lines))
 
+    def _package_index_file(self, save_dir):
+        """Convert and save index file with package."""
+        e_license = ['/*', '\n'.join(' * ' + line.lstrip('# ') for line in LICENSE_HEADER.splitlines()), ' */', '']
+        comments = [
+            '// Auto generated file from scripts/regen_prepackage_rules_index.sh',
+            '// Do not hand edit. Run that script to regenerate package information instead',
+            ''
+        ]
+        rule_imports = [f"import rule{i} from './{os.path.splitext(os.path.basename(r.path))[0] + '.json'}';"
+                        for i, r in enumerate(self.rules, 1)]
+        const_exports = ['export const rawRules = ['] + [f"  rule{i}," for i in range(1, len(self.rules))] + ['];', '']
+
+        index_ts = e_license + comments + rule_imports + const_exports
+        with open(os.path.join(save_dir, 'index.ts'), 'w') as f:
```

Comment:
_explicit is better than implicit_
```suggestion
        with open(os.path.join(save_dir, 'index.ts'), 'wt') as f:
```

Reply:
lol - you are all over that `t` today